src/LiveComponent/src/Resources/doc/index.rst

```diff
@@ -1204,6 +1204,7 @@ via the ``LiveCollectionType``::
     use Symfony\Component\Form\FormBuilderInterface;
     use Symfony\Component\OptionsResolver\OptionsResolver;
     use Symfony\UX\LiveComponent\Form\Type\LiveCollectionType;
+    use Symfony\UX\LiveComponent\Attribute\LiveProp;
```

Comment:
Think this should be in the next code block, no?

Reply:
yes you are right